TESTS/network/emac/README.md

```diff
@@ -8,21 +8,71 @@ A device can be configured to be a CTP echo server by enabling `echo-server` set
 
 # Test cases
 
-## EMAC interface initialise
+## EMAC initialise
 
-Initializes EMAC interface driver.
+Test case initializes the EMAC driver and test network stack.
 
-For WLAN installs test case so that it can intercept incoming Ethernet messages from the WLAN driver. Incoming CTP frames are handed by the test case and other frames are forwarded to the LWIP stack.
+EMAC test environment uses test network stack as the default network stack. Stack is enabled by setting
+the `nsapi.default-stack` option in test environment's application `json` file to value `TEST`.
```

Comment:
- The EMAC... as the default stack (I think you don't need to repeat the network)
- To enable the stack, set the...